File: MyList.Domain/Common/Models/ContentModels/Game.cs
using System.ComponentModel.DataAnnotations;
using MyList.Domain.Common.Models.Tags;

namespace MyList.Domain.Common.Models.ContentModels
{
    public class Game
    {
        [Key]
        public Guid GameID { get; set; }
        public string Name { get; set; }
        public string? Description { get; set; }
        public string? PictureURL { get; set; }
        public ICollection<GameTag>? Tags { get; set; } = new List<GameTag>();
        public float? UserScore { get; set; } //enum
        public float? GlobalScore { get; set; } //enum
        public string? UserStatus { get; set; } //enum
        public string? GlobalStatus { get; set; } //enum
        public string? GameStudio { get; set; }
        public DateTime? RelizeDate { get; set; }
        public Guid? ApplicationUserId { get; set; }
    }
}

File: src/MyList.UI/src/app/models/Game.ts
import { Data } from "@angular/router";
import { Tag } from "./TagDto";

export interface Game{
    GameID :string;
    Name: string;
    Description: string;
    PictureURL:string;
    Tags: Tag[];
    UserScore: number;
    GlobalScore: number;
    UserStatus: string;
    GlobalStatus: string;
    GameStudio : string;
    RelizeData: Data;
    ApplicationUserId:string;
}